components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -322,10 +322,21 @@ def run_clp(
         if last_archive_stats is not None and (
             None is stats or stats["id"] != last_archive_stats["id"]
         ):
+            archive_id = last_archive_stats["id"]
+            archive_path = archive_output_dir / archive_id
+            if StorageEngine.CLP_S == clp_storage_engine:
+                metadata_uploader_cmd = [
+                    str(clp_home / "bin" / "indexer"),
+                    "--db-config-file",
+                    str(db_config_file_path),
+                    "default",  # hardcode the table name for now
```

Comment:
I see why you want to have support for multiple tables but I don't think it makes sense to enforce that within the package since the package *doesn't* currently support multiple tables. Instead, I would suggest we:

* leave the column metadata table name as `{table_prefix}column_metadata`.
  * When we eventually add support for multiple tables/datasets in the clp package, 
we can modify this.
* modify the indexer process to take in the *full* table name rather than just the part after "column_metadata_". That way, we can easily control what the table should be called and there's less hard-coding of constants.

Reply:
Based on our discussion offline, we decided not to do this but comment about why we need a table name.